politeiad/backendv2/tstorebe/tstore/tlogclient.go

```diff
@@ -121,23 +118,14 @@ type tclient struct {
 func (t *tclient) TreeNew() (*trillian.Tree, *trillian.SignedLogRoot, error) {
 	log.Tracef("trillian TreeNew")
 
-	pk, err := ptypes.MarshalAny(t.privateKey)
-	if err != nil {
-		return nil, nil, err
-	}
-
 	// Create new trillian tree
 	tree, err := t.admin.CreateTree(t.ctx, &trillian.CreateTreeRequest{
 		Tree: &trillian.Tree{
-			TreeState:          trillian.TreeState_ACTIVE,
-			TreeType:           trillian.TreeType_LOG,
-			HashStrategy:       trillian.HashStrategy_RFC6962_SHA256,
-			HashAlgorithm:      sigpb.DigitallySigned_SHA256,
-			SignatureAlgorithm: sigpb.DigitallySigned_ED25519,
-			DisplayName:        "",
-			Description:        "",
-			MaxRootDuration:    ptypes.DurationProto(0),
-			PrivateKey:         pk,
```

Comment:
Same comment.  If the private key is not being set here anymore, where is it being set?